model/src/main/kotlin/OrtResult.kt

```diff
@@ -441,6 +441,16 @@ data class OrtResult(
             emptyList()
         }
 
+    /**
+     * Retrieve all unresolved and non-excluded issues with severities equal to or over [minSeverity].
+     */
+    @JsonIgnore
+    fun getOpenIssues(minSeverity: Severity = Severity.WARNING) = collectIssues()
+        .filterNot { (id, _) -> isExcluded(id) }
+        .values
```

Comment:
These could be combined via

    .mapNotNull { (id, issues) -> issues.takeUnless { isExcluded(id) } }